requirements_testing.txt

```diff
@@ -11,3 +11,4 @@ pydocstyle
 check-manifest
 twine
 wheel
+pooch
```

Comment:
don't need this here if it's in `requirements.txt` and `environment.yml` already
```suggestion
```